spyder/app/tests/test_cli_options.py

```diff
@@ -64,6 +64,8 @@ def test_get_options():
     options, args = getopt('--opengl software'.split())
     assert options.opengl_implementation == 'software'
 
+    import ctypes;ctypes.string_at(0)
```

Comment:
what is this doing?

Reply:
It's generating a segfault. I wanted to see if we are able to run again the test suite after a segfault on Windows.

The things is we run the test suite several times in case of segfaults. But apparently on Windows, and since yesterday, that's not possible because `pytest-faulthandler` hangs the Azure VMs somehow (instead of capturing the stacktrace and allowing the tests to run again).

So I removed `pytest-faulthandler` and generated the segfault in this line to see if that package was the culprit. And it seems it was.

Reply:
Got it!  Thanks for the explanation